packages/perseus/src/util.graphie.ts

```diff
@@ -0,0 +1,57 @@
+const svgLabelsRegex = /^web\+graphie:/;
+// For offline exercises in the mobile app, we download the graphie data
+// (svgs and localized data files) and serve them from the local file
+// system (with file://). We replace urls that start with `web+graphie`
+// in the perseus json with this `file+graphie` prefix to indicate that
+// they should have the `file://` protocol instead of `https://`.
+const svgLocalLabelsRegex = /^file\+graphie:/;
+
+// Sometimes other components want to download the actual image e.g. to
+// determine its size. Here, we transform an .svg-labels url into the
+// correct image url, and leave normal image urls alone
+export function getRealImageUrl(url: string): string {
+    if (isLabeledSVG(url)) {
+        return getSvgUrl(url);
+    }
+    return url;
+}
+
+export function isLabeledSVG(url: string): boolean {
```

Comment:
I'm not requesting a change, but wanted to get your thoughts. The mixture of terms in this area are confusing unless you know Graphie. I wonder if we should refer to everything as a Graphie image instead of sometimes using "Labeled SVG" and sometimes Graphie.